host_applications/linux/apps/raspicam/raspiraw.c

```diff
@@ -676,8 +676,8 @@ int main(int argc, const char** argv) {
 
 	if(cfg.exposure_us != -1)
 	{
-		cfg.exposure = (int)(cfg.exposure_us / sensor_mode->line_time);
-		vcos_log_error("Setting exposure to %d from time %d", cfg.exposure, cfg.exposure_us);
+		cfg.exposure = (int)(cfg.exposure_us * 1000 / sensor_mode->line_time_ns);
```

Comment:
You no longer need the cast - all variables involved are already ints.
I did give you a duff pointer though - the *1000 has the potential to overflow an int, so you need to cast cfg.exposure_us to int64_t (or long long int) before the multiply.

I've fixed it up, squashed the commits, and merged it all manually. Thanks.